### PR TITLE
JBIDE-26291

### PR DESCRIPTION
**Pull request by vprusa**
Fixed some errors with Maven update call,
added as repository https://coderplus.github.io/m2e-connector-for-jaxws-maven-plugin/ because the old one had broken one of the underlying xml files,
updated versions - bundles, eap-7.x,
ignore known warning.

In progress, will add test run soon.

### CODE REVIEW

File: tests/org.jboss.tools.examples.ui.bot.test/src/org/jboss/tools/examples/ui/bot/test/integration/AbstractImportQuickstartsTest.java [hunk 1]
```diff
@@ -437,6 +441,25 @@ protected static void deleteAllProjects() {
 	protected static void cleanupShells() {
 		WorkbenchShellHandler.getInstance().closeAllNonWorbenchShells();
 	}
+	
+	protected void mavenUpdate(Quickstart quickstart) {
```
Comment (vprusa) [inline tests/org.jboss.tools.examples.ui.bot.test/src/org/jboss/tools/examples/ui/bot/test/integration/AbstractImportQuickstartsTest.java]: Eh, I just noticed, I need to change this nested try catches to ifs if possible..

**Comment by vprusa** [comment]
Test runs:
[eap](https://dev-platform-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/examples.eap.it.weekly/465/)
[wildfly](https://dev-platform-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/examples.wildfly.it.weekly/416/)
Errors are caused by ```Exception during sync execution in UI thread```  and misspelled project names. 
javaee fails because of different issue - failing on time exceeded and memory problems.

**Comment by jkopriva** [comment]
I was looking at error "Exception during sync execution in UI thread" and it always happens on the same line. If you look inside stacktrace, there is an exception: "Caused by: org.eclipse.swt.SWTException: Widget is disposed". This is a common issue and you can look here how to fix it: https://github.com/eclipse/reddeer/issues/817

**Comment by vprusa** [comment]
Seems like an issue for yet another PR.

**Comment by jkopriva** [comment]
@vprusa ok then, please create a JIRA for it and linked it and I will merge this one.

**Comment by jkopriva** [comment]
testPR

**Comment by jkopriva** [comment]
testPR

**Comment by jkopriva** [comment]
testPR

**Comment by jkopriva** [comment]
testPR

**Comment by jkopriva** [comment]
testPR

**Comment by jkopriva** [comment]
testPR

**Comment by jkopriva** [comment]
JIRA: https://issues.jboss.org/browse/JBIDE-26291

**Comment by jkopriva** [comment]
JIRA: https://issues.jboss.org/browse/JBIDE-26291

**Pull request merged**